.github/workflows/test.yml

```diff
@@ -0,0 +1,30 @@
+name: Go Unit Tests
+
+on:
+  push:
```

Comment:
Do we need to run this when we merge to master? I think it should be enough to only run on PR

Reply:
Not sure if we have push to master enabled or not

Reply:
I'll add a branch block as soon as this is merged! There is also a setting to require latest code before merge. Thing is with running this on merge to master is no one will see it

Reply:
okok, changed it now